src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  createArray() {
+    if (!Animal.alive) {
+      Animal.alive = [];
+    }
+  }
+  // The ways with static alive = []
+  // or Animal.alive = [] didnt pass the Elinter;
```

Comment:
Which line you add? It should be fine. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review